pages/about.jsx

```diff
@@ -7,79 +7,107 @@ var Illustration = require('../components/illustration.jsx');
 
 var config = require('../config/config');
 
-var Intro = (
-  <div>
-    <h1>About the Mozilla Learning Network</h1>
-    <section className="intro">
-      <Illustration width={226} height={226}
-      src1x="/img/pages/about/hive-nyc-summer-quest.jpg"
-      alt="HiveLearningNYC 2014 Hive NYC Summer Quest Maker Party">
-        <p>
-          We are cultivating a global network of web literacy leaders who are teaching and
-          learning the most important skills of our age: the ability to read, write and
-          participate in the digital world. Our programs and resources can help you make
-          an impact in your community, and participate in an openly networked world.
-        </p>
-      </Illustration>
-    </section>
-  </div>
-);
+var Intro = React.createClass({
```

Comment:
I don't want to have over 1000 lines changes in this patch, so follow up coming soon :)